src/ui/public/accessibility/kbn_ui_ace_keyboard_mode.less

```diff
@@ -0,0 +1,24 @@
+@import (reference) "~ui/styles/variables";
+
+.ui-ace-keyboard-hint {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  left: 0;
+  background: rgba(255, 255, 255, 0.7);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  opacity: 0;
+}
+
+.ui-ace-keyboard-hint-inactive {
+  display: none;
+}
+
+.ui-ace-keyboard-hint:focus {
+  opacity: 1;
+  border: 2px solid @globalColorBlue;
+  z-index: 1000;
+}
```

Comment:
I will reformat them to use camel case syntax. Why would nesting the states be more efficient? In CSS the `:focus` will anyway compile to the same, so no difference there. The `isInactive` state should actually be slower, since it has to check multiple classes instead of a single on on the element. Though since we have the example for states like this in our styleguide I will of course also fix it (though in my opinion giving states the full prefix (`uiAceKeyboardHint-`), makes nesting kind of unnecessary, but I won't fight the styleguide here :-)